lib/login/accountCreation.dart

```diff
@@ -0,0 +1,216 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+
+import 'package:http/http.dart' as http;
+import 'package:off_top_mobile/login/login.dart';
+
+int repoNumber;
+
+class SignUp extends StatefulWidget {
+  const SignUp({@required this.email});
+  final String email;
+  @override
+  _SignUpState createState() => _SignUpState();
+}
+
+class _SignUpState extends State<SignUp> {
+  // Getting value from TextField widget.
+  //final idController = TextEditingController();
+  final TextEditingController firstNameController = TextEditingController();
+  final TextEditingController lastNameController = TextEditingController();
+  final TextEditingController cityController = TextEditingController();
+  final TextEditingController ageController = TextEditingController();
+  final TextEditingController genderController = TextEditingController();
+  final TextEditingController professionalController = TextEditingController();
+  final TextEditingController usernameController = TextEditingController();
+
+  bool visible = false;
+  Future<void> makePostRequest() async {
+    setState(() {
+      visible = true;
+    });
+    debugPrint('In postrequest');
+
+    // Getting value from Controller
+    //String Id = idController.text;
+    final String firstName = firstNameController.text;
+    final String lastName = lastNameController.text;
+    final String city = cityController.text;
+    final String age = ageController.text;
+    final String gender = genderController.text;
+    final String professional = professionalController.text;
+    final String username = usernameController.text;
+    final DateTime now = DateTime.now();
+    final String formatDate =
+        '${now.month.toString()}/${now.day.toString()}/${now.year.toString()}';
+    //String email = emailController.text;
+
+    // API
+    const String address = 'http://localhost:9000/setUser';
+    final Map<String, String> headers = <String, String>{
+      'Content-type': 'application/json'
+    };
+    final dynamic data = {
+      'id': 1,
+      'age': age,
+      'city': city,
+      'firstName': firstName,
+      'lastName': lastName,
+      'gender': gender,
+      'professional': professional,
+      'email': widget.email,
+      'username': username,
+      'password': 'HoldTheDoor',
+      'createdAt': formatDate,
+      'deletedAt': 'null',
+    };
```

Comment:
This is some good stuff. I like it

Reply:
Can we make this a User Object and put it in mode/User.dart

Reply:
@frlzjosh User Object has now been created under User.dart.